modules/gitops-accessing-gitops-monitoring-dashboards.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/monitoring-with-gitops-dashboards.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-accessing-gitops-monitoring-dashboards_{context}"]
+= Accessing {gitops-shortname} monitoring dashboards
+
+The monitoring dashboards are deployed automatically by the Operator. You can access {gitops-shortname} monitoring dashboards from the *Administrator* perspective of the {OCP} web console. 
+
+[NOTE]
+====
+Disabling or changing the content of the dashboards is not supported.
+====
+
+.Prerequisites
+
+* You have access to the {OCP} web console.
+* The {gitops-title} Operator is installed in the default namespace, `openshift-gitops-operator`.
+* The cluster monitoring is enabled on the `openshift-gitops-operator` namespace.
```

Comment:
@ekristova Add this before the last bullet item: OpenShift Container Platform monitoring is installed on the cluster.

Reply:
@Srivaralakshmi In our documentation, it is mentioned that the monitoring is preinstalled, this is why I did not include it as a prerequisite: https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html#about-openshift-monitoring (it is here in the first sentence)
So do you think it should be a part of the prerequisite anyway?